### ISSUE TITLE
Patient profiles: can't add must support to Patient.deceased.deceasedDateTime.date-accuracy-indicator

### ISSUE THREAD

**Issue by dtr-agency** [AuDigitalHealth/ci-fhir-stu3#17]
# Prerequisites
- [x] I have verified the problem exists in the available content
- [x] I have searched [open](https://github.com/ci-fhir-stu3/Issues/issues) and [closed](https://github.com/ci-fhir-stu3/Issues/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aclosed) issues to make sure it isn't already reported
- [x] I have written a descriptive issue title

# The bug
Patient.deceased.deceasedDateTime.date-accuracy-indicator should have must support flagged. However if the element has a must support flag set then they can't be opened in FORGE. 

This applies to all of the profiles of Patient:
* [Base Patient] (https://github.com/AuDigitalHealth/ci-fhir-stu3/blob/master/resources/patient-dh-base-1.xml)
* [Patient with Mandatory Identifier] (https://github.com/AuDigitalHealth/ci-fhir-stu3/blob/master/resources/patient-ident-1.xml)
* [Patient with Mandatory IHI] (https://github.com/AuDigitalHealth/ci-fhir-stu3/blob/master/resources/patient-ihi-1.xml)

## What I expected to happen
The Patient.deceased.deceasedDateTime.date-accuracy-indicator element should be in the Differential table with a must support flag. 

## Steps to reproduce
1. Open a patient profile
2. set mustSupport to "true"
3. Save and try to open in FORGE
4. See that the profile will not open

example xml fragment showing the code that causes the issue
```
<element id="Patient.deceased[x]:deceasedDateTime.extension:accuracyIndicator">
    <path value="Patient.deceasedDateTime.extension"/>
     <sliceName value="accuracyIndicator"/>
       <type>
         <code value="Extension"/>
         <profile value="http://hl7.org.au/fhir/StructureDefinition/date-accuracy-indicator"/>
       </type>
       <mustSupport value="true"/>
 </element>
```

## QA.html excerpt
N/A

## Available workarounds
1. At this time a known issue and commenting out that content allows the profile to be edited with FORGE. 
2. The second option (not currently in play) is to have a continuous manual edit cycle of comment out to open in FORGE, edit in FORGE, and then comment in to have it show up in the generated output. 

## Desktop (please complete the following information):
- OS: Windows 10
- Browser: Chrome
- Date this bug was detected: 06 April 2018

## Additional context
https://chat.fhir.org/login/#narrow/stream/implementers/topic/Forge.20error.20when.20adding.20Must.20support.20.3D.20true.20to.20an.20extension
https://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemEdit&tracker_item_id=17165

**Comment by dtr-agency** [comment]
Backlog story: https://jira.digitalhealth.gov.au/browse/CIFMM-1265

**Comment by dtr-agency** [comment]
This work has been addressed via commit a3f794c
